### PR TITLE
Add resource_estimate integration test

### PR DESCRIPTION
**Pull request by paaige**

### CODE REVIEW

File: cirq_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import cirq
 import pytest
 from applications_superstaq import SuperstaQException
+from applications_superstaq import ResourceEstimate
```
Comment (singular-value) [inline cirq_superstaq/daily_integration_test.py:9]: alphabetize and merge

File: cirq_superstaq/daily_integration_test.py [hunk 2]
```diff
@@ -75,6 +76,27 @@ def test_get_balance(service: css.Service) -> None:
...
+
+    circuit1 = cirq.Circuit(cirq.H(q0), cirq.CNOT(q0, q1), cirq.measure(q0))
+
+    resource_estimate = service.resource_estimate(circuit1, "neutral_atom_qpu")
+
+    assert resource_estimate == ResourceEstimate(2, 1, 3)
+
+    q2 = cirq.LineQubit(0)
+    q3 = cirq.LineQubit(1)
+    circuit2 = cirq.Circuit(cirq.H(q2), cirq.CZ(q2, q3), cirq.CNOT(q2, q3), cirq.measure(q3))
```
Comment (singular-value) [inline cirq_superstaq/daily_integration_test.py:91]: use the same circuit as in qss? In particular, CNOT first, then CZ.

also, why not stick with q0 and q1?

**Review by singular-value — CHANGES_REQUESTED** [review]
couple comments, also still failing tests

**Comment by singular-value** [comment]
The checks seem to be related to the release of a new cirq version and the deprecation of some previous cirq features.

**Review by singular-value — APPROVED** [review]
lgtm, please check locally before deploying.

**Review by vtomole — APPROVED** [review]

**Pull request merged**